dataset-processor-python/data_loader.py

```diff
@@ -107,74 +72,67 @@ def convert_str_to_num(value):
             # if not number, return the original value
             return value
 
+    def compile_cell_feature_analysis(
```

Comment:
Nit: I think `add` might be a better word than `compile` here, since `compile` to me suggests that calling this function should be a final step and something should be returned?